src/framework/preferences/js/Panels.js

```diff
@@ -692,6 +692,93 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         }
     });
 
+    /********************************************************
+     * A base grade for styled radio Button adjuster panels *
+     ********************************************************/
+
+    fluid.defaults("fluid.prefs.panel.styledRadioButtonAdjuster", {
+        gradeNames: ["fluid.prefs.panel"],
+        mergePolicy: {
+            "controlValues.theme": "replace",
```

Comment:
I'm not good at coming up with names. What's in my mind is:

- The component name: radioButtonInPreview
- Remove the use of `theme` in the component
- Improve the comment to explain the style of each radio button is a preview of what gets apply by corresponding enactor. The current wording of `styled radio button` seems not quite relevant with radio buttons being previews.